helm-org.el

```diff
@@ -202,95 +202,32 @@ Note: [1] A separator can be a comma, a colon i.e. [,:] or a space.
-              (if parents (nreverse cands) cands))))))
-
-(defun helm-source-org-headings-for-files (filenames &optional parents)
-  "Build source for org headings in files FILENAMES.
-When PARENTS is specified, bild source for heading that are parents of
-current heading."
-  (helm-make-source "Org Headings" 'helm-org-headings-class
-    :filtered-candidate-transformer 'helm-org-startup-visibility
-    :parents parents
-    :candidates filenames))
-
-(defun helm-org-startup-visibility (candidates _source)
-  "Indent headings and hide leading stars displayed in the helm buffer.
-If `org-startup-indented' and `org-hide-leading-stars' are nil, do
-nothing to CANDIDATES."
-  (cl-loop for i in candidates
-	   collect
-           ;; Transformation is not needed if these variables are t.
-	   (if (or helm-org-show-filename helm-org-format-outline-path)
-	       (cons
-		(car i) (cdr i))
-             (cons
-              (if helm-org-headings-fontify
-                  (when (string-match "^\\(\\**\\)\\(\\* \\)\\(.*\n?\\)" (car i))
-                    (replace-match "\\1\\2\\3" t nil (car i)))
-                (when (string-match "^\\(\\**\\)\\(\\* \\)\\(.*\n?\\)" (car i))
-                  (let ((foreground (org-find-invisible-foreground)))
-                    (with-helm-current-buffer
-                      (cond
-                       ;; org-startup-indented is t, and org-hide-leading-stars is t
-                       ;; Or: #+STARTUP: indent hidestars
-                       ((and org-startup-indented org-hide-leading-stars)
-                        (with-helm-buffer
-                          (require 'org-indent)
-                          (org-indent-mode 1)
-                          (replace-match
-                           (format "%s\\2\\3"
-                                   (propertize (replace-match "\\1" t nil (car i))
-                                               'face `(:foreground ,foreground)))
-                           t nil (car i))))
-                       ;; org-startup-indented is nil, org-hide-leading-stars is t
-                       ;; Or: #+STARTUP: noindent hidestars
-                       ((and (not org-startup-indented) org-hide-leading-stars)
-                        (with-helm-buffer
-                          (replace-match
-                           (format "%s\\2\\3"
-                                   (propertize (replace-match "\\1" t nil (car i))
-                                               'face `(:foreground ,foreground)))
-                           t nil (car i))))
-                       ;; org-startup-indented is nil, and org-hide-leading-stars is nil
-                       ;; Or: #+STARTUP: noindent showstars
-                       (t
-                        (with-helm-buffer
-                          (replace-match "\\1\\2\\3" t nil (car i)))))))))
-              (cdr i)))))
-
-(defun helm-org-get-candidates (filenames &optional parents)
```

Comment:
@thierryvolpiatto  Spacemacs used this function https://github.com/syl20bnr/spacemacs/issues/13123  

Would you have a suggestion what function can be used instead? :thinking: